streams/src/commonTest/kotlin/com/mirego/trikot/streams/reactive/processors/CombineLatestProcessorTests.kt

```diff
@@ -159,4 +159,59 @@ class CombineLatestProcessorTests {
 
         assertEquals(1, errorCount)
     }
+
+    @Test
+    fun whenUsingSafeCombineNoValueAreDispatchedWhenMissing() {
+        val firstPublisher = MockPublisher("a")
+        val secondPublisher = MockPublisher()
+        var firstValueReceived: String? = null
+        var secondValueReceived: String? = null
+
+        firstPublisher.safeCombine(secondPublisher)
+            .subscribe(CancellableManager()) { (value1, value2) ->
+                firstValueReceived = value1
+                secondValueReceived = value2
+            }
+
+        assertNull(firstValueReceived)
+        assertNull(secondValueReceived)
```

Comment:
This test could be better. Here we want to validate that combine didn't dispatch but it may did dispatch with null value and the result would be the same.

What do you think of this instead?

```suggestion
        var didDispatch: Boolean = false

        firstPublisher.safeCombine(secondPublisher)
            .subscribe(CancellableManager()) { (value1, value2) ->
                didDispatch = true
            }

        assertFalse(didDispatch)
```